core/consensus/common/src/import_queue.rs

```diff
@@ -627,7 +633,7 @@ mod tests {
 		assert_eq!(link_port.recv(), Ok(LinkMsg::BlockImported));
 
 		// Send an unknown
-		let results = vec![(Ok(BlockImportResult::ImportedUnknown(Default::default(), Default::default())), Default::default())];
+		let results = vec![(Ok(BlockImportResult::ImportedUnknown(Default::default(), Default::default(), None)), Default::default())];
```

Comment:
Perhaps one could add a testcase for the `Some(who)` case as well? It cover asserting having first received the `LinkMsg::BlockImported` message, followed by the the `LinkMsg::Disconnected`.